diffs/heartbeat.diff

```diff
@@ -0,0 +1,1933 @@
+            </a>
+            <code class="literal">
+             write.codec
+@@ -598,7 +598,7 @@
+            .
+           </p>
+           <h5>
+-           <a id="_literal_write_flush_timeout_literal">
++           <a id="_write_flush_timeout">
+            </a>
+            <code class="literal">
+             write.flush.timeout
+@@ -622,7 +622,7 @@
+            The default value is 1s.
+           </p>
+           <h5>
+-           <a id="_literal_write_flush_events_literal">
++           <a id="_write_flush_events">
+            </a>
+            <code class="literal">
+             write.flush.events
+@@ -638,7 +638,7 @@
+            The default value is 16384.
+           </p>
+           <h5>
+-           <a id="_literal_read_flush_timeout_literal">
++           <a id="_read_flush_timeout">
+            </a>
+            <code class="literal">
+             read.flush.timeout
+html/en/beats/heartbeat/master/configuring-output.html
+html/en/beats/heartbeat/master/configuring-ssl-logstash.html
+html/en/beats/heartbeat/master/connection-problem.html
+html/en/beats/heartbeat/master/console-output.html
+--- /dev/fd/63
++++ html/en/beats/heartbeat/master/console-output.html
+@@ -210,7 +210,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_pretty_literal">
++                <a id="_pretty">
+                 </a>
+                 <code class="literal">
+                  pretty
+@@ -235,7 +235,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_codec_literal_4">
++                <a id="_codec_4">
+                 </a>
+                 <code class="literal">
+                  codec
+@@ -271,7 +271,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_enabled_literal_6">
++                <a id="_enabled_6">
+                 </a>
+                 <code class="literal">
+                  enabled
+@@ -295,7 +295,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_bulk_max_size_literal_5">
++                <a id="_bulk_max_size_5">
+                 </a>
+                 <code class="literal">
+                  bulk_max_size
+html/en/beats/heartbeat/master/contributing-to-beats.html
+html/en/beats/heartbeat/master/convert.html
+html/en/beats/heartbeat/master/decode-base64-field.html
+html/en/beats/heartbeat/master/decode-json-fields.html
+html/en/beats/heartbeat/master/decompress-gzip-field.html
+html/en/beats/heartbeat/master/defining-processors.html
+--- /dev/fd/63
++++ html/en/beats/heartbeat/master/defining-processors.html
+@@ -306,20 +306,24 @@
+               At the top-level in the configuration. The processor is applied to all data collected by Heartbeat.
+              </li>
+              <li class="listitem">
+-              <p class="simpara">

```

Comment:
fixed in https://github.com/elastic/beats/commit/94215011793fe59b9088ce29f2c5eb9881dc1952